src/main/kotlin/de/livepoll/api/service/PollService.kt

```diff
@@ -123,15 +155,28 @@ class PollService(
                 this.startDate = null
                 this.endDate = null
             }
-            if (poll.slug != null && isSlugUnique(poll.slug)) {
-                this.slug = poll.slug
+            if (poll.slug != null && !this.slug.equals(poll.slug)) {
```

Comment:
How is this done here? Does the poll get saved when I do not change the slug, but change the name for example. Looks like the poll would not get updated when the slug hasn't changed.

Reply:
If the slug has not been changed the if block is skipped and at the poll is saved at the end of the update poll method. The new name would be saved

Reply:
Ah, okay. Thanks for the clarification